src/frontend/src/app/AppAuthenticated.tsx

```diff
@@ -62,32 +68,36 @@ const AppAuthenticated: React.FC<AppAuthenticatedProps> = ({ userId, userRole })
           height: '100vh',
           position: 'fixed',
           width: 15,
-          borderRight: 2,
+          borderRight: isGuestOnMemberPage ? 2 : 0,
           borderRightColor: theme.palette.background.paper
         }}
       />
-      <IconButton
-        onClick={() => {
-          setDrawerOpen(true);
-          setMoveContent(true);
-        }}
-        sx={{ position: 'fixed', left: -8, top: '3%' }}
-      >
-        <ArrowCircleRightTwoToneIcon
-          sx={{
-            fontSize: '30px',
-            zIndex: 1,
-            '& path:first-of-type': { color: '#000000' },
-            '& path:last-of-type': { color: '#ef4345' }
-          }}
-        />
-      </IconButton>
-      <Sidebar
-        drawerOpen={drawerOpen}
-        setDrawerOpen={setDrawerOpen}
-        moveContent={moveContent}
-        setMoveContent={setMoveContent}
-      />
+      {isGuestOnMemberPage && (
```

Comment:
Perhaps a better way to do this would be to have a single usestate which is whether you are on the guest home page or not. I'll give a think if i can think of a better way than that